scapy/contrib/vqp.py

```diff
@@ -54,10 +54,8 @@ class VQPEntry(Packet):
         FieldLenField("len", None),
         ConditionalField(IPField("datatom", "0.0.0.0"),
                          lambda p: p.datatype == 3073),
-        ConditionalField(MACField("data", "00:00:00:00:00:00"),
-                         lambda p: p.datatype == 3078),
-        ConditionalField(MACField("data", "00:00:00:00:00:00"),
-                         lambda p: p.datatype == 3080),
+        ConditionalField(MACField("MAC_data", "00:00:00:00:00:00"),
```

Comment:
Why change the name?

Reply:
There's another field right after that named "data" for entries without MAC addresses.

Reply:
In that case use a `MultipleTypeField`